sdk/src/AbstractMain.cpp

```diff
@@ -199,13 +199,15 @@ int AbstractMain::run(int in_argc, char** in_argv)
    addLogDestination(
```

Comment:
Let's replace this entire log destination with the FileLogDestination we are creating below. Basically, you should be able to cut that code and paste it over this `addLogDestination`.